assertj-core/src/main/java/org/assertj/core/api/AbstractDateAssert.java

```diff
@@ -27,6 +27,7 @@
 import static org.assertj.core.util.Lists.list;
 import static org.assertj.core.util.Lists.newArrayList;
 
+import java.sql.Timestamp;
```

Comment:
This probably has an underlying impact on the module descriptor as `java.sql` would become required whenever an assertion with an `Instant` parameter is called.

I'll see how we can prove it (related to #2759).

Reply:
I'm not ready yet with #2759 and #3442, which would help demonstrate whether we have a dependency. I'd go ahead with the merge now and fix it later if something pops up.